torchgeo/datasets/zuericrop.py

```diff
@@ -56,9 +58,13 @@ class ZueriCrop(VisionDataset):
     md5s = ["1635231df67f3d25f4f1e62c98e221a4", "5118398c7a5bbc246f5f6bb35d8d529b"]
     filenames = ["ZueriCrop.hdf5", "labels.csv"]
 
+    band_names = tuple(["NIR", "B03", "B02", "B04", "B05", "B06", "B07", "B11", "B12"])
+    RGB_BANDS = ["B04", "B03", "B02"]
+
     def __init__(
         self,
         root: str = "data",
+        bands: Tuple[str, ...] = band_names,
```

Comment:
Needs to be added to docstring

Reply:
Also, I don't think it has to be a tuple, any Sequence should work right?

Reply:
based on #289 it should be a tuple, I thought.

Reply:
The only thing in #289 I see is:

> I like list or tuple of string band names

Lists and tuples are both subsets of `typing.Sequence`